BT-Tracking/AppDelegate.swift

```diff
@@ -297,4 +315,18 @@ private extension AppDelegate {
         KeychainService.BUID = nil
         KeychainService.TUIDs = nil
     }
+
+    private func isDeviceSupported() -> Bool {
+        #if targetEnvironment(simulator)
+        return true
+        #else
+        let model = UIDevice.current.modelName
+        if model.hasPrefix("iPhone") {
```

Comment:
Na iPadu a iPodu to muze bezet taky :). 

Asi bych to udelal pres https://github.com/devicekit/DeviceKit, a udelel array devicu ktere nemuzou mit iOS 13. A proti tomu to checkoval. (vsechny nezname budou povolene)